src/pe/mod.rs

```diff
@@ -923,4 +923,10 @@ mod tests {
             "Permissive parsing should succeed for packed binary"
         );
     }
+
+    #[test]
+    fn malformed_resource_tree() {
+        let data = include_bytes!("../../tests/bins/pe/malformed_resource_tree.exe.bin");
+        let _ = PE::parse(data);
+    }
```

Comment:
This being succeeding though is a malformed binary. What is the intention of this test?